Multiverse-io/companies_service
# Adjust RabbitMQ setup

### What does this PR do?
This is adding a few fixes and adjustments to the RabbitMQ setup:

- Adding the config for demo/staging in `runtime.exs`
- Renamed routing keys and changed the dummy publisher into a real company [changelog publisher](https://coda.io/d/_daQaM_fX8Uq/Writing-async-message-pipelines_su3S5#_lu0Gz) (but with just an ID for now)

### How to test
1. Run `mix setup` to set the application up and create the RabbitMQ exchange locally.
2. Run `iex -S mix phx.server` to run the server in interactive mode
3. Run something like `CompaniesService.Publishers.CompanyChangelog.publish(%{id: 1})` in the iex console
4. Check if both the event publication log and the event consuming log are displayed

#### Links
[Ticket](https://app.shortcut.com/multiverse/story/45579/add-rabbitmq-setup-to-companies-service)
